server/src/test/java/org/elasticsearch/search/functionscore/FunctionScoreIT.java

```diff
@@ -83,7 +83,7 @@
             scripts.put("doc['random_score']", vars -> {
                 Map<?, ?> doc = (Map) vars.get("doc");
                 ScriptDocValues.Doubles randomScore = (ScriptDocValues.Doubles) doc.get("random_score");
-                return randomScore.getValue();
+                if (randomScore != null) {return randomScore.getValue();} else return 0;
```

Comment:
If this is going to be on a single line, please use a ternary statement. The same applies to the other tests here that were changed in a similar way.